platform/ios/PolyPodApp/PolyPod/AppDelegate.swift

```diff
@@ -11,7 +11,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         Log.bootstrap()
         Log.info("Application initialized")
-        
+
+        UserDefaults.standard.disableDataProtection()
```

Comment:
Nit: We're creating a reference to `UserDefaults.standard` just below, so why not just call `defaults.disableDataProtection()` a bit lower?

Reply:
yeah, could be. `standard` is anyway static, but to be consistent I will update it.